projects/1.20/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2465 @@
+  "tetra.stats.reap_revenge.tooltip_extended": "最近攻击过你的实体被判定为复仇目标，持有具有复仇能力的物品并注视复仇目标时会出现一个HUD指示器，使用复仇能力会重置目标的复仇状态。所有复仇目标在30秒内未被攻击后复仇状态将重置",
+  "tetra.stats.ability_echo": "回声",
+  "tetra.stats.lunge_echo.tooltip": "在半空中时，猛冲可以额外使用§e%s§r次，以延长距离或改变方向",
+  "tetra.stats.lunge_echo.tooltip_extended": "在半空中按Shift/潜行键来向后闪避， 按空格/跳跃键（不是右键）来增加向上移动的距离",
+  "tetra.stats.execute_echo.tooltip": "3秒后再次攻击目标，造成基于其已损失生命值和负面效果的伤害",
+  "tetra.stats.slam_echo.tooltip": "3秒后对同一目标二次猛击",
+  "tetra.stats.puncture_echo.tooltip": "3秒后再次刺穿目标",
+  "tetra.stats.pry_echo.tooltip": "3秒后再次撬击目标",
+  "tetra.stats.overpower_echo.tooltip": "持有者的精疲力竭延迟§e%s§r秒生效，再次压制则进一步延迟",
+  "tetra.stats.reap_echo.tooltip": "3秒后对同一区域二次收割\n \n§r首次击杀伤害§e+1§r，持续§e30§r秒，后续击杀伤害§e+%s§r",
+  "tetra.stats.execute": "处决",
+  "tetra.stats.execute.tooltip": "按住右键可以以§e100%%（%s）§f的基础伤害发动一次强力攻击，且伤害会因以下因素进一步提升：\n§7目标每损失1%%的生命值，伤害§e+%s%%§7\n目标每带有1层负面效果，伤害§e+%s%%§7\n \n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
+  "tetra.stats.execute.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.severing": "切割",
+  "tetra.stats.severing.tooltip": "攻击有§e%s%%§r概率使目标最大生命值§e-10%%§r，攻击伤害§e-5%%§r，持续§e60§r秒，此效果最多叠加§e%s§r层",
+  "tetra.stats.lunge": "猛冲",
+  "tetra.stats.lunge.tooltip": "按住右键蓄力，松开施展猛冲。猛冲击中实体时，会造成§e%s%%（%s）§f的伤害并使能力冷却时间§e-30%%§f\n \n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
+  "tetra.stats.lunge.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.slam": "猛击",
+  "tetra.stats.slam.tooltip": "按住右键蓄力，松开释放强力猛击\n \n§7猛击地面会从撞击点发出冲击波，被冲击波击中的实体受到§e%s%%（%s）§7的伤害\n \n§7猛击的命中目标会受到§e%s%%（%s）§7的伤害并被强力击退，同时使能力冷却时间§e+50%%§7\n \n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
+  "tetra.stats.slam.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.puncture": "刺穿",
+  "tetra.stats.puncture.tooltip": "按住右键蓄力，松开发动可以刺穿目标盔甲的攻击，造成§e100%%（%s）§f的伤害并使目标护甲值§e-%s%%§f，持续§e%s§f秒\n \n§7如果刺穿一个护甲值少于§e6§7点或者已经被刺穿的目标，则会造成共计§e8§7点流血伤害，持续§e4§7秒\n \n§7能力冷却时间随目标护甲值增加而增加，每点护甲值§e+0.5§7秒\n \n§7蓄力时间：§e%s§7秒\n§7基础冷却时间：§e%s§7秒",
+  "tetra.stats.puncture.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.overpower": "压制",
+  "tetra.stats.overpower.tooltip": "按住右键蓄力，可以发动一次造成§e%s%%（%s）§f伤害但几乎不造成击退的攻击\n \n§7给予目标§e2§7层精疲力竭，给予攻击者§e1§7层。每层使移动速度§e-10%%§7，攻击速度§e-5%%§7。即使攻击失败，攻击者也会精疲力竭\n \n§7精疲力竭持续时间：§e%s§7秒\n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
+  "tetra.stats.overpower.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.reap": "收割",
+  "tetra.stats.reap.tooltip": "按住右键蓄力，松开发动横扫攻击，对范围内实体造成§e%s%%（%s）§f的伤害\n \n§7效果范围：§e%s§7格\n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
+  "tetra.stats.reap.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.pry_armor": "撬击",
+  "tetra.stats.pry_armor.tooltip": "右击1个实体造成§e50%%（%s）§f的伤害并使其护甲值§e-%s§f，持续§e%s§f秒，减甲效果可以叠加\n \n§7冷却时间：§e%s§7秒",
+  "tetra.stats.pry_armor.tooltip_extended": "冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Cooldown is based on the item's swing speed. Bonus damage from effects like smite is added to the base damage of the attack. The duration of the armor reduction effect is refreshed every time it is applied.**

```suggestion
  "tetra.stats.pry_armor.tooltip_extended": "冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
```